src/EpisodeAction.cpp

```diff
@@ -118,7 +118,7 @@ bool EpisodeActionPrivate::parse( const QVariant& data )
     if( episodeActionMap.contains( QLatin1String( "timestamp" ) ) )
     {
         s = episodeActionMap.value( QLatin1String( "timestamp" ) );
-        m_timestamp = s.toULongLong();
+        m_timestamp = static_cast<qulonglong>(QDateTime::fromString(s.toString(), QStringLiteral("yyyy-MM-ddThh:mm:ss")).toSecsSinceEpoch());
```

Comment:
FYI:
QDateTime::toSecsSinceEpoch() was introduced in Qt 5.8, so the current required Qt version needs to be increased, and Qt 4 dropped.

Reply:
Fair point.
I've changed the implementation to QDateTime::toMSecsSinceEpoch() which was introduced in Qt 4.8.